### DISCUSSION TITLE
Motiongenstatusget invalid start state joint

### DISCUSSION THREAD

**Discussion by 6413365-chinnagrit** [NVlabs/curobo#428] [Software Q&A]
Hi, I was trying to config the new robot and follow the tutorial. In the end i got stuck with press play and robot go to weird posture and not moving with the error of  Plan did not converge to a solution: MotionGenStatus.INVALID_START_STATE_JOINT_LIMITS

what should i do to fix this ?. Thanks you in advance
![Screenshot from 2024-11-01 15-34-16](https://github.com/user-attachments/assets/4f348497-c413-4c7b-adc3-8861043fbbdc)
![Screenshot from 2024-11-01 15-37-13](https://github.com/user-attachments/assets/e5bde89b-4def-4338-b9be-a2914751f70b)

**Comment by balakumar-s** [comment]
Can you load the robot into sim without issues? Try importing the urdf into Isaac Sim and clicking play (without using cuRobo's examples). If you are unable to load the robot into isaac sim, please ask questions here: https://forums.developer.nvidia.com/c/omniverse/simulation/69
